handsontable/src/3rdparty/walkontable/src/renderer/table.js

```diff
@@ -297,14 +297,17 @@ export default class TableRenderer {
     // Fix for multi-line content and for supporting `rowHeights` option.
     for (let visibleRowIndex = 0; visibleRowIndex < rowsToRender; visibleRowIndex++) {
       const TR = rows.getRenderedNode(visibleRowIndex);
+      const rowUtils = this.rowUtils;
 
       if (TR.firstChild) {
         const sourceRowIndex = this.renderedRowToSource(visibleRowIndex);
-        const rowHeight = this.rowUtils.getHeightByOverlayName(sourceRowIndex, this.activeOverlayName);
+        const rowHeight = rowUtils.getHeightByOverlayName(sourceRowIndex, this.activeOverlayName);
+        const isBorderBoxSizing = rowUtils.dataAccessObject.stylesHandler.areCellsBorderBox();
```

Comment:
You should not use the `dataAccessObject` of the `rowUtils` module as it's its "private" DAO. I'd recommend injecting the StylesHandler separately into the Renderer module.